src/portoctl.cpp

```diff
@@ -688,15 +693,15 @@ class TEnterCmd : public ICmd {
         return TError(EError::Unknown, "Can't find root for " + subsys);
     }
 
-    int Execute(int argc, char *argv[]) {
+    int Execute(TCommandEnviroment *env) final override {
         bool enterCgroups = true;
-        int start = GetOpt(argc, argv, {
+        const auto &args = env->GetOpts({
             { 'C', false, [&](const char *arg) { enterCgroups = false; } },
         });
 
-        string cmd = "";
-        for (int i = start + 1; i < argc; i++) {
-            cmd += argv[i];
+        string cmd;
+        for (size_t i = 1; i < args.size(); ++i) {
```

Comment:
All these changes should be reviewed very carefully.
Are there tests for portoctl functionality ?
All of portoctl command are classes local to portoctl.cpp file and cannot be tested with unit-tests.
Maybe we can have some kind of script to run portoctl comands and check some expects?